Sources/Sentry/SentrySessionTaskSearch.m

```diff
@@ -0,0 +1,57 @@
+/**
+ * Part of this code was copied from
+ * https://github.com/AFNetworking/AFNetworking/blob/4eaec5b586ddd897ebeda896e332a62a9fdab818/AFNetworking/AFURLSessionManager.m#L407-L418
+ * under the MIT license
+ */
+
+#import "SentrySessionTaskSearch.h"
+#import <objc/runtime.h>
+
+@implementation SentrySessionTaskSearch
+
++ (NSArray<Class> *)urkSessionTaskClassesToTrack
```

Comment:
`h`: I think pointing out that this can break easily is important.
```suggestion
/**
* WARNING: This code can easily lead to bad behavior, like crashes or messing up HTTP requests. Be careful when changing it.
*/
+ (NSArray<Class> *)urkSessionTaskClassesToTrack
```